client-v2/src/selectors/collectionArticleSelectors.js

```diff
@@ -0,0 +1,38 @@
+// @flow
+
+import { createSelector } from 'reselect';
+
+import type { State } from '../types/State';
+import type { CollectionArticles } from '../types/Collection';
+import type { CapiArticle } from '../types/Capi';
+
+const allCollectionArticlesSelector = (
+  state: State
+): { [string]: CollectionArticles } => state.collectionArticles;
+
+const collectionIdSelector = (state: State, id: string) => id;
+
+const articleStageSelector = (state: State, id: string, stage: string) => stage;
+
+const getCollectionArticles = (
+  collectionArticles: { [string]: CollectionArticles },
+  id: string,
+  stage: string
+): Array<CapiArticle> => {
+  if (!collectionArticles) {
```

Comment:
This can be combined with the check below
```javascript
if (!collectionArticles || !collectionArticles[id]) {
  return [];
}
```

or, if you like a ternary, you can combine all 3:
```javascript
return !collectionArticles || !collectionArticles[id] ? [] : collectionArticles[id][stage];
```